### DISCUSSION TITLE
在预测时少量预测结果出现为空的现象，准确率为0.0

### DISCUSSION THREAD

**Discussion by TyrionChou** [PaddlePaddle/PaddleOCR#13301] [Q&A]
#### 问题描述 / Problem Description
自己训练的crnn文本识别模型，在预测时少量预测结果出现为空的现象，准确率为0.0，因为词典中没有空的字符，所以很奇怪

#### 运行环境 / Runtime Environment
- OS:ubuntu cuda11.1 cudnn8.0
- Paddle:2.7.0.2
- PaddleOCR:2.3.2.post111

**Comment by GreatV** [comment]
是不是数据量不够

**Comment by TyrionChou** [comment]
> 是不是数据量不够

预测的结果是blank，怀疑是因为词表中第一个字符读取的时候多了一个\uffef导致的
